setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+all_args=$@
+echo $all_args > /tmp/test_file11
+bucket=hailstone-ami-scan-results
+cd /usr/local/bin
+PATH=${PATH}:/usr/local/bin
+echo "after path setup"
+#sleep
```

Comment:
I think we should remove commented out code like the 2 sleep commands and the wget command below.